scripts/deploy_staking_escrow.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+from brownie import StakingEscrow, config
+from scripts.utils import CURRENT_NETWORK, LOCAL_BLOCKCHAIN_ENVIRONMENTS, get_account
+
+
+def deploy_mocks():
+    """This function should deploy nucypher_token and t_staking and return the
+    corresponding contract addresses"""
+    nucypher_token = None
+    t_staking = None
+    return nucypher_token, t_staking
+
+
+def main(id=None):
```

Comment:
`id` is shadowing the built-in `id` function, and it's not immediately clear what's the ID for. What about `account_id`?